.circleci/config.yml

```diff
@@ -553,26 +557,72 @@ jobs:
           name: Deploy to gh-pages branch
           command: |
             gh-pages --dotfiles --message "[skip ci] documentation update" --dist target/doc
-
+  premainnet-cluster-test:
+    executor: premainnet-cluster-test-executor
+    description: Run cluster test against Premainnet
+    steps:
+      - run:
+          name: Set Premainnet Clustetest Environment
+          command: |
+            # git revision in premainnet can be found on this site https://fburl.com/cynwp2sv:
```

Comment:
I'd suggest remove it here. Only those on intern can access.  In its place, this workflow can be referenced in the run book we run the perf test on pre-mainnet.

Reply:
Runbook has been updated and referenced.